src/mixins/focus-in.js

```diff
@@ -3,8 +3,7 @@ import { eventOff, eventOn } from '../utils/dom'
 export default {
   data () {
     return {
-      focusInElement: null,
```

Comment:
Just thinking, for future reference to future coders, it might be good to create this non-reactive property during the `beforeCreate()` hook:

```js
  beforeCreate () {
    // Create non-reactive property
    this.focusInElement = null
  }
```

Reply:
I think once we add this to the mixin, all should be good to merge.